variable.go

```diff
@@ -433,11 +434,33 @@ func (vr *variableResolver) resolve(ctx *ExecutionContext) (*Value, error) {
 				currArgs = append([]functionCallArgument{executionCtxEval{}}, currArgs...)
 			}
 
+			// If kwargs map is needed
+			includeKwargsBit := 0
```

Comment:
define includeKwargsBit as a bool may be clearer

Reply:
Need it as a int because I need to subtract 1 from some places if kwargs map is present.